src/resources/lang/enso/ro.json

```diff
@@ -1,6 +1,151 @@
 {
-    "Key": null,
+    "Die Aktion war erfolgreich": null,
+    "App": null,
     "Value": null,
+    "Key": null,
+    "Core": null,
+    "Only missing": null,
+    "edit texts": null,
+    "keys found": null,
+    "German": null,
+    "Choose language": null,
+    "Edit Tutorial": null,
+    "Error": null,
+    "Postal Area": null,
+    "Localisation Index": null,
+    "Time": null,
+    "Imported Entries": null,
+    "Import Summary": null,
+    "File": null,
+    "Line": null,
+    "Delete Template": null,
+    "Download Template": null,
+    "Import": null,
+    "Upload Template": null,
+    "Imported At": null,
+    "Imported By": null,
+    "File name": null,
+    "Successful": null,
+    "Issues": null,
+    "ImportType": null,
+    "Past Imports": null,
+    "data import": null,
+    "The tutorial was created!": null,
+    "Permission": null,
+    "Tutorials Index": null,
+    "The log was cleaned": null,
+    "The log file": null,
+    "Current file size is": null,
+    "was last updated": null,
+    "Show Log": null,
+    "MB": null,
+    "Last updated": null,
+    "Size": null,
+    "The role was created!": null,
+    "Roles Index": null,
+    "The menu was created!": null,
+    "Menus Index": null,
+    "The permissions were created!": null,
+    ".selectOptions": null,
```

Comment:
we don't want starting from here

Reply:
?

Reply:
I remove the .xxx again but if we collect again they will be added again

Reply:
this is what I meant by "the flag shoud be used with care" :)